server/test/repository/AccountRepositoryTest.java

```diff
@@ -75,32 +76,35 @@ public void updateClientNameTest_AllNonEmpty() {
     ApplicantModel applicantUpdateTest = setupApplicantForUpdateTest();
     AccountModel account = setupAccountForUpdateTest();
     repo.updateTiClient(
-        account, applicantUpdateTest, "Dow", "James", "John", "", "", "", "2020-10-10");
+        account, applicantUpdateTest, "Dow", "James", "John", "Jr.", "", "", "", "2020-10-10");
     assertThat(applicantUpdateTest.getApplicantData().getApplicantFirstName().get())
         .isEqualTo("Dow");
     assertThat(applicantUpdateTest.getApplicantData().getApplicantMiddleName().get())
         .isEqualTo("James");
     assertThat(applicantUpdateTest.getApplicantData().getApplicantLastName().get())
         .isEqualTo("John");
+    assertThat(applicantUpdateTest.getApplicantData().getApplicantNameSuffix().get())
+        .isEqualTo("Jr.");
   }
 
   @Test
   public void updateClientNameTest_EmptyMiddleAndLastName() {
```

Comment:
**suggestion:** Let's update the test name to indicate suffix is empty too?

Reply:
yup updated test name.